minicluster/src/main/java/tachyon/master/LocalTachyonClusterMultiMaster.java

```diff
@@ -109,7 +109,6 @@ public synchronized TachyonFileSystem getClient() throws IOException {
     return mClientPool.getClient(mMasterConf);
   }
 
-  // TODO(cc): Since we have MasterContext now, remove this.
   public TachyonConf getMasterTachyonConf() {
```

Comment:
Both this and the same one in `LocalTachyonCluster` can be removed, but I thought it's fine to leave them here since a lot of tests are using them...